app/src/event_converter.c

```diff
@@ -102,6 +102,23 @@ convert_keycode(SDL_Keycode from, enum android_keycode *to, uint16_t mod,
     if (mod & (KMOD_LALT | KMOD_RALT | KMOD_LGUI | KMOD_RGUI)) {
         return false;
     }
+
+    if (!(mod & (KMOD_NUM | KMOD_SHIFT))) {
+    	//	handling Numpad events when Num Lock is disabled
+        switch(from){
```

Comment:
I believe that there are some coding style issues.

Reply:
I did some modifications regarding the coding style issues. Can you tell me what was exactly the issue?

Thank you

Reply:
Great! From now, I only notice that we need a space after switch.